addons/mail/static/src/core/common/message.scss

```diff
@@ -30,9 +30,11 @@
 
 .o-mail-Message-body {
     a.o_mail_redirect {
-        padding: map-get($spacers, 1);
-        border-radius: $border-radius;
-        background: rgba($primary, .2);
+        background: rgba($primary, .2) !important;
+    }
+
+    a.o_channel_redirect {
+        background: rgba($primary, .2) !important;
```

Comment:
This is redundant with `a.o_mail_redirect` and `a.o_channel_redirect` style rules in `composer.scss`.
Note that this can be moved in `mail/core.scss`